src/assets/translations/main.json

```diff
@@ -50,8 +50,7 @@
     "apy": "%{apy}% APY",
     "settings": "Settings",
     "legalDated": "Dated: January 2022",
-    "approveOnWallet": "Approve On Wallet",
-    "confirmOnWallet": "Confirm On Wallet"
+    "approve": "Approve",
```

Comment:
You'll need to remove this comma - no trailing comma in `json`.

Reply:
whoops